src/test/unit/mocks/SJPMockPage.json

```diff
@@ -0,0 +1 @@
+{"document":{"publicationDate":"dolor proident sit","version":"3h;R(p1","documentName":"liv]"},"courtLists":[{"courtHouse":{"courtRoom":[{"session":[{"sittings":[{"hearing":[{"party":[{"partyRole":"PROSECUTOR","individualDetails":{"individualForenames":"Danny","individualSurname":"Thomas","address":{"town":"Swansea"}},"organisationDetails":{"organisationName":"qU8QlEo@"}},{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Nigel","individualSurname":"Sausage","address":{"postCode":"G14 9DU"},"individualMiddleName":"F","title":"Miss"}}],"offence":{"offenceTitle":"Dreadful singing and ambiguous ticket writing","reportingRestriction":false}}]},{"hearing":[{"party":[{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Christopher","individualSurname":"Robin","address":{"county":"London","postCode":"E15 3AR","town":"London"},"title":"Mr","individualMiddleName":"W"},"organisationDetails":{"organisationName":"Ministry of Justice"}},{"partyRole":"PROSECUTOR","individualDetails":{"individualForenames":"Junaid","individualSurname":"Thomas","address":{"town":"Swansea"}},"organisationDetails":{"organisationName":"qU8QlEo@"}}],"offence":{"offenceTitle":"Lollipop overindulgence","reportingRestriction":false}}]}]}]}]}}]}
```

Comment:
please for the love of all that is tidy in the world, prettify and remove from the 1 line
https://jsonformatter.curiousconcept.com/#